staging/src/k8s.io/apiserver/pkg/admission/initializer/initializer.go

```diff
@@ -52,6 +59,13 @@ func (i pluginInitializer) Initialize(plugin admission.Interface) {
 	if wants, ok := plugin.(WantsAuthorizer); ok {
 		wants.SetAuthorizer(i.authorizer)
 	}
+
+	if wants, ok := plugin.(WantsClientCert); ok {
+		if i.proxyClientCert == nil || i.proxyClientKey == nil {
+			panic("An admission plugin wants a client cert/key, but they were not provided.")
```

Comment:
Shouldn't panic here.  The validator for the admission plugin can decide how to handle it.  If the admission plugin fails to validate, we fail.

Reply:
fair enough, wrong assumption - copy, paste existing code.